src/battleground/Battleground.js

```diff
@@ -12,13 +12,22 @@ import GameObject from './GameObject.js';
 import { getTestTank } from '../tanks/TankLoader.js';
 import { verifyLogin } from '../globalComponents/verifyLogin.js';
 
+type Props = {|
+	setPlayerOneTankName: (string) => void,
```

Comment:
Battleground shouldn't be setting these manually. All Battleground should need to do is pass the Tank objects to the healthbar, and the healthbar should be able to render based on those tank objects.